src/Data/Fin/Properties.agda

```diff
@@ -761,6 +767,23 @@ pinch-mono-≤ 0F      {suc j} {suc k} (s≤s j≤k) = j≤k
 pinch-mono-≤ (suc i) {0F}    {k}     0≤n       = z≤n
 pinch-mono-≤ (suc i) {suc j} {suc k} (s≤s j≤k) = s≤s (pinch-mono-≤ i j≤k)
 
+pinch-injective :
+  {k : ℕ} {x : Fin k} {y z : Fin (ℕ.suc k)} →
+  suc x ≢ y → suc x ≢ z → pinch x y ≡ pinch x z → y ≡ z
+pinch-injective {y = zero} {zero} f g p = refl
```

Comment:
Can we give more meaningful names to `f` and `g`, e.g. `1+x≢y`? And primary equalities like `p` are usually called `eq`...